src/Kalkulator/Kalkulator.tsx

```diff
@@ -32,7 +32,8 @@ const Kalkulator: FunctionComponent<Props> = props => {
 
     useEffect(() => {
         if (defaultTapteDagsverk.status === RestStatus.Suksess && !harEndretTapteDagsverk) {
```

Comment:
Trenger litt feilhåndtering for å hindre evig spinn hvis kallet ikke returnerer 200